pkg/handler/login_confirm.go

```diff
@@ -35,8 +35,7 @@ func (l *LoginReviewHandler) GetTokenInfo() model.ConnectTokenInfo {
 
 func (l *LoginReviewHandler) WaitReview(ctx context.Context) (bool, error) {
 	lang := i18n.NewLang(l.i18nLang)
-	vt := term.NewTerminal(l.readWriter, "y/n: ")
-	utils.IgnoreErrWriteString(vt, lang.T("Need ACL review, continue? (y/n): "))
+	vt := term.NewTerminal(l.readWriter, lang.T("Need ACL review, continue? (y/n): "))
 	utils.IgnoreErrWriteString(vt, utils.CharNewLine)
 	count := 0
 	for {
```

Comment:
There aren't many changes to the code mentioned above since it does not look like there are any specific syntax errors, logical discrepancies or inefficiencies that could be identified. If you're referring to some other aspect of the code or application, please describe more accurately for help with optimizations or checks.